freqtrade/optimize/backtesting.py

```diff
@@ -113,15 +113,11 @@ def backtest(args) -> DataFrame:
     max_open_trades = args.get('max_open_trades', 0)
     realistic = args.get('realistic', True)
     record = args.get('record', None)
+    exchange_name = args.get('exchange_name', 'bittrex')
```

Comment:
Do we need a default value at Bittrex?

Reply:
I didn't want to change default behaviour where only bittrex was supported, but I can change it None